### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by lippserd**

### CODE REVIEW

File: doc/02-Installation.md [hunk 1]
```diff
@@ -1,49 +1,261 @@
...
+* PHP (≥7.2)
+* MySQL or PostgreSQL PDO PHP libraries
+* The following PHP modules must be installed: `cURL`, `dom`, `json`, `libxml`
+* [Icinga DB](https://github.com/Icinga/icingadb)
+* [Icinga Web 2](https://github.com/Icinga/icingaweb2) (≥2.9)
+* [Icinga PHP Library (ipl)](https://github.com/Icinga/icinga-php-library) (≥0.9)
+* [Icinga PHP Thirdparty](https://github.com/Icinga/icinga-php-thirdparty) (≥0.11)
+* For exports to PDF the [pdfexport](https://github.com/Icinga/icingaweb2-module-pdfexport) (≥0.10)
+  module is required (Optional)
+<!-- {% endif %} -->
```
[the remark this answers is not in the document]
Comment (lippserd) [reply doc/02-Installation.md:261]: It's from the `if $index ... $else ...`.

**Review by nilmerg — COMMENTED** [review]
* [x] The migration chapter still uses the blacklist term
* [x] There are migration/upgrade steps missing for the black->deny change (rc2 users may have those configured)

File: doc/02-Installation.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (nilmerg) [reply doc/02-Installation.md:261]: Okay, but then the from source needs its own main chapter. It's at least how it's for Icinga Web 2.

File: doc/02-Installation.md [hunk 2]
```diff
@@ -1,49 +1,269 @@
...
+[how to install modules](https://icinga.com/docs/icinga-web-2/latest/doc/08-Modules/#installation) from source.
+Make sure you use `icingadb` as the module name. The following requirements must also be met.
+
+### Requirements
+
+* PHP (≥7.2)
+* MySQL or PostgreSQL PDO PHP libraries
+* The following PHP modules must be installed: `cURL`, `dom`, `json`, `libxml`
+* [Icinga DB](https://github.com/Icinga/icingadb)
+* [Icinga Web 2](https://github.com/Icinga/icingaweb2) (≥2.10)
```
Comment (nilmerg) [inline doc/02-Installation.md:206]: Why have you changed this to 2.10? We should be compatible with 2.9 still.

**Review by nilmerg — APPROVED** [review]

**Pull request merged**